aws-xray/src/main/java/io/opentelemetry/contrib/awsxray/AwsXrayRemoteSamplerProvider.java

```diff
@@ -41,22 +34,4 @@ public Sampler createSampler(ConfigProperties config) {
   public String getName() {
     return "xray";
   }
-
-  // Currently the only way to read the Resource from autoconfiguration. Best would be if the SPI
-  // could return a Function<SamplerFactoryArgs, Sampler> where SamplerFactoryArgs has
-  // SDK-constructed components like Resource and Clock.
-  @AutoService(AutoConfigurationCustomizerProvider.class)
-  public static final class ResourceHolder implements AutoConfigurationCustomizerProvider {
```

Comment:
is it ok to delete this public class given `aws-xray` module is stable?

Reply:
thanks, how about mark it `@Deprecated` and make it a no-op, since that will still conform to our versioning guarantees for stable artifacts: https://github.com/open-telemetry/opentelemetry-java/blob/main/VERSIONING.md#compatibility-requirements

Reply:
Makes sense, I'll make the changes today.